docs/data/material/getting-started/templates/sign-in/SignIn.tsx

```diff
@@ -39,73 +107,223 @@ export default function SignIn() {
     });
   };
 
+  const validateInputs = () => {
+    const email = document.getElementById('email') as HTMLInputElement;
+    const password = document.getElementById('password') as HTMLInputElement;
+
+    let isValid = true;
+
+    if (!email.value || !/\S+@\S+\.\S+/.test(email.value)) {
+      setEmailError(true);
+      setEmailErrorMessage('Please enter a valid email address.');
+      isValid = false;
+    } else {
+      setEmailError(false);
+      setEmailErrorMessage('');
+    }
+
+    if (!password.value || password.value.length < 6) {
+      setPasswordError(true);
+      setPasswordErrorMessage('Password must be at least 6 characters long.');
+      isValid = false;
+    } else {
+      setPasswordError(false);
+      setPasswordErrorMessage('');
+    }
+
+    return isValid;
+  };
+
   return (
-    <ThemeProvider theme={defaultTheme}>
-      <Container component="main" maxWidth="xs">
-        <CssBaseline />
-        <Box
+    <ThemeProvider theme={showCustomTheme ? SignInTheme : defaultTheme}>
+      <CssBaseline />
+      <Stack
+        direction={{ xs: 'column', sm: 'row' }}
+        justifyContent="space-between"
+        sx={(theme) => ({
+          backgroundImage:
+            theme.palette.mode === 'light'
+              ? `linear-gradient(180deg, ${alpha('#CEE5FD', 0.7)}, #FFF)`
+              : `linear-gradient(${alpha('#02294F', 0.5)}, ${alpha('#021F3B', 0.0)})`,
+          backgroundRepeat: 'no-repeat',
+          px: { xs: 2, sm: 4 },
+          height: '100dvh',
+        })}
+        component="main"
+      >
+        <Stack
+          direction="row"
+          justifyContent="space-between"
+          sx={{ display: { xs: 'flex', sm: 'none' }, my: 4 }}
+        >
+          <Button
+            startIcon={<ArrowBackRoundedIcon />}
+            component="a"
+            href="/material-ui/getting-started/templates/landing-page/"
+          >
+            Back
+          </Button>
+          <ToggleColorMode mode={mode} toggleColorMode={toggleColorMode} />
+        </Stack>
+        <Button
+          startIcon={<ArrowBackRoundedIcon />}
+          component="a"
+          href="/material-ui/getting-started/templates/landing-page/"
```

Comment:
I agree with it! When I was designing it, very often I click on the back button, expecting it to take me back to the Templates page. However, this could be a problem for users coming from other templates—IMO even in such cases, linking back to the Templates page wouldn't be a significant issue. Therefore, I'm down for linking it back to the Templates page.